sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1242,6 +1242,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val REMOVE_REDUNDANT_SORTS_ENABLED = buildConf("spark.sql.execution.removeRedundantSorts")
+    .internal()
+    .doc("Whether to remove redundant physical sort node")
+    .version("3.1.0")
```

Comment:
If we backport this into branch-2.4, `3.1.0` -> `2.4.8`?

Reply:
I am not exactly sure if it's better to change it in this PR or to change it when this PR is backported to 2.4.8.